src/overlays/actors/ovl_Boss_Tw/z_boss_tw.h

```diff
@@ -6,9 +6,132 @@
 
 struct BossTw;
 
+typedef void (*BossTW_ActionFunc)(struct BossTw* this, GlobalContext* globalCtx);
+
+enum Work {
+    /*  0 */ CS_TIMER_1,
+    /*  1 */ CS_TIMER_2,
+    /*  2 */ TW_PLLR_IDX,
+    /*  3 */ TAIL_IDX,
+    /*  4 */ BLINK_IDX,
+    /*  5 */ INVINC_TIMER,
+    /*  6 */ FOG_TIMER,
+    /*  7 */ CAN_SHOOT,
+    /*  8 */ UNK_S8,
+    /*  9 */ TW_BLINK_IDX,
+    /* 10 */ YAW_TGT,
+    /* 11 */ PLAYED_CHRG_SFX,
+    /* 12 */ BURN_TMR,
+    /* 12 */ WORK_MAX = BURN_TMR
```

Comment:
```suggestion
    /* 12 */ BURN_TMR,
    /* 13 */ WORK_MAX
```
`BURN_TMR` is used so I'm not sure why work_max is BURN_TMR

`/* 0x0168 */ char unused_168[0x10];` then needs to become `/* 0x0170 */ char unused_170[0xE];`